.github/linters/coplanning/utils.py

```diff
@@ -72,11 +72,14 @@ def make_request(
         with urllib.request.urlopen(req) as response:
             return json.loads(response.read().decode())
     except urllib.request.HTTPError as e:
-        err(f"HTTP request failed: {e.code} - {e.reason}")
+        err(f"HTTP request failed: {e.code} - {e.reason}", exit=True)
+        return {}  # For type checking, never reached due to sys.exit() in err()
```

Comment:
What if we split the case where `err` should exit into a separate function like:
```py
def err(message: str) -> None:
    logger.error(message)

def err_and_exit(message: str) -> typing.Never:
    err(message)
    sys.exit(1)
```
Can add a return of `typing.Never` on the one that exits so the type-checker knows that function won't return so you don't need the `return {}` here.